recipes/libcusolvermp/meta.yaml

```
@@ -0,0 +1,118 @@
+{% set version = "0.5.1.690" %}
+{% set libcal_version = "0.4.4.50" %}
+{% set lib_version = version.split(".")[0]+"."+version.split(".")[1]+"."+version.split(".")[2]+".0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-sbsa" %}    # [aarch64]
+{% set extension = "tar.xz" %}
+
+{% set cuda_major = environ.get("cuda_compiler_version", "12.6").split(".")[0] %}
+
+package:
+  name: libcusolvermp
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cusolvermp/redist/libcusolvermp/{{ platform }}/libcusolvermp-{{ platform }}-{{ version }}_cuda{{ cuda_major }}-archive.{{ extension }}
+  sha256: ea12f900a6a01e5ed6d4907d15e01ed017c903d9fae4c0da5798a7e71c90db66  # [linux64 and (cuda_compiler_version or "").startswith("12")]
+  sha256: 16799e30ff56e1eac31e35a4e94bc0971ef98b6352c9f278fd28954e384c7181  # [aarch64 and (cuda_compiler_version or "").startswith("12")]
+
+build:
+  number: 0
+  skip: true  # [not (linux64 or aarch64)]
+  skip: true  # [cuda_compiler_version in (None, "None") or (cuda_compiler_version or "").startswith("11")]
+  script:
+    - cp -rv include $PREFIX
+    - cp -rv lib $PREFIX
+
+test:
+  commands:
+    - test -L $PREFIX/lib/libcusolverMp.so.0
+    - test -f $PREFIX/lib/libcusolverMp.so.{{ lib_version }}
+
+outputs:
+  - name: libcusolvermp
+    build:
+      run_exports:
+        - {{ pin_subpackage("libcusolvermp", max_pin="x") }}
+    files:
+      - lib/libcusolverMp.so.*
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ compiler('cuda') }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - {{ stdlib("c") }}
+      host:
+        - cuda-version {{ cuda_compiler_version }}
+        - libcusolver    # [(cuda_compiler_version or "").startswith("12")]
+        - cuda-cudart    # [(cuda_compiler_version or "").startswith("12")]
+        - libcublas      # [(cuda_compiler_version or "").startswith("12")]
+        - libcal {{ libcal_version }}
+      run:
+        - libcusolver    # [(cuda_compiler_version or "").startswith("12")]
+        - cuda-cudart    # [(cuda_compiler_version or "").startswith("12")]
+        - libcublas      # [(cuda_compiler_version or "").startswith("12")]
```

Since we only support CUDA12 does it make sense to clean this up?
```suggestion
        - libcusolver
        - cuda-cudart
        - libcublas
        - libcal {{ libcal_version }}
      run:
        - libcusolver
        - cuda-cudart
        - libcublas
```

---

Makes sense. I will remove those.